FossClubWebsite/WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't want these .xml files. Try to write in simple Angular right now.

Reply:
but if we use eclipse then these gets included by default i think. Then what should i do to remove these things